addon/components/listbox/-option.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { guidFor } from '@ember/object/internals';
+
+export default class ListboxOptionComponent extends Component {
+  @tracked guid = `${guidFor(this)}-headlessui-listbox`;
```

Comment:
`@tracked` can be removed here, since the value will not change after being set in the constructor for this class

Reply:
It seems like we only ever reference this `guid` property by appending `-option` to the end of it; would it make more sense to define the `guid` property with that `-option` suffix and avoid needing to remember that each location needs to have the suffix appended manually?

Reply:
https://github.com/GavinJoyce/ember-headlessui/pull/76/commits/2c8811db9b9f3a0ea02136f36418607a18748d85 cleans this up, so that the `guid` always has the correct suffix.